test/deposit.e2e-spec.ts

```diff
@@ -0,0 +1,86 @@
+import request from "supertest";
+import { INestApplication, ValidationPipe } from "@nestjs/common";
```

Comment:
Probably it won't impact the tests results, but the app uses [this validation pipe](https://github.com/across-protocol/scraper-api/blob/stage/src/validation.pipe.ts), not the default one. We should replace it in order to fully replicate the production scenario

Reply:
Ah ok will change